app/models/creator_setting.rb

```diff
@@ -0,0 +1,7 @@
+class CreatorSetting < ApplicationRecord
+  # This class exists to take advantage of Rolify for limiting authorization
+  # on internal reports.
+  # NOTE: It is not backed by a database table and should not be expected to
```

Comment:
If it's not backed by a table, it probably shouldn't inherit from `ApplicationRecord` since accessing this model in any way will raise an exception:

```
(pry) output error: #<ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "creator_settings" does not exist
LINE 8:  WHERE a.attrelid = '"creator_settings"'::regclass
```